modules/shared/src/main/scala/org/tessellation/security/Hash.scala

```diff
@@ -20,6 +20,7 @@ object hash {
   @newtype
   case class Hash(value: String) {
     def getBytes: Array[Byte] = value.getBytes(StandardCharsets.UTF_8)
+    def shortValue: String = s"Hash[${value.take(5)}...${value.takeRight(5)}]"
```

Comment:
I see you use it for debugging only, so IMO it should be a `Show` instance

Reply:
@marcinwadon There is already a `Show` defined for this class. How do you want the 2nd one defined?

Reply:
In the object, you can create another show instance like that:

```scala
val shortShow: Show[Hash] = hash => s"Hash(value=${hash.value.take(5)}...${hash.value.takeRight(5)})"
```

and at usage time, you can assign it to implicit value:
```scala
implicit val shortHashShow: Show[Hash] = Hash.shortShow
```
so then it takes priority over the one available in the case class

Reply:
because you use it exactly as a `Show`, so let's use standarized ways rather than crafting custom ones for something simple